freqtrade/exchange/binance.py

```diff
@@ -136,17 +150,22 @@ def fill_leverage_brackets(self):
                 else:
                     leverage_brackets = self._api.load_leverage_brackets()
 
-                for pair, brackets in leverage_brackets.items():
-                    self._leverage_brackets[pair] = [
-                        [
-                            min_amount,
-                            float(margin_req)
-                        ] for [
-                            min_amount,
-                            margin_req
-                        ] in brackets
-                    ]
-
+                for pair, brkts in leverage_brackets.items():
+                    [amt, old_ratio] = [0.0, 0.0]
+                    brackets = []
+                    for [notional_floor, mm_ratio] in brkts:
+                        amt = (
+                            (
+                                (float(notional_floor) * (float(mm_ratio) - float(old_ratio)))
+                            ) + amt
```

Comment:
I feel like there's "some" level of too many parenteses in this segment (at least one)...